InventoryBundleProduct/Plugin/InventorySales/IsBundleProductSalablePlugin.php

```diff
@@ -0,0 +1,110 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+declare(strict_types=1);
+
+namespace Magento\InventoryBundleProduct\Plugin\InventorySales;
+
+use Magento\Bundle\Model\Product\Type;
+use Magento\Catalog\Api\ProductRepositoryInterface;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\InventoryBundleProduct\Model\GetBundleProductStockStatus;
+use Magento\InventoryCatalogApi\Model\GetProductTypesBySkusInterface;
+use Magento\InventorySales\Model\IsProductSalableCondition\IsProductSalableConditionChain;
+
+/**
+ * Check if bundle product is salable with bundle options.
+ */
+class IsBundleProductSalablePlugin
```

Comment:
The namespace already includes `Plugin`.
Yes, we already have a bunch of plugins in the Inventory codebase with a similar naming and it is not described in the technical guidelines, but please take attention anyway.